ar_marker_id: 0
ar_topic: "/model_tracker/ar_model_pose"